packages/core-react/src/eip6963.ts

```diff
@@ -0,0 +1,3 @@
+import { createStore } from 'mipd';
+
+export const providersStore = createStore();
```

Comment:
Can this conflict with wagmi instance?

Reply:
Built-in into wagmi mipd store is disabled by `multiInjectedProviderDiscovery: false` option in the wagmi config